mkosi/__init__.py

```diff
@@ -352,9 +362,12 @@ def finalize_scripts(scripts: Mapping[str, Sequence[PathString]] = {}) -> Iterat
         yield Path(d)
 
 
-def finalize_host_scripts(state: MkosiState, chroot: Sequence[PathString]) -> contextlib.AbstractContextManager[Path]:
+def finalize_host_scripts(
+        state: MkosiState,
+        helpers: Mapping[str, Sequence[PathString]],
```

Comment:
indented too far
```suggestion
    state: MkosiState,
    helpers: Mapping[str, Sequence[PathString]],
```

You might want `(setq 'python-indent-def-block-scale 1)`—solved that issue for me.